client/app/validations/submittable-pas-form.js

```diff
@@ -0,0 +1,58 @@
+import SaveablePasForm from './saveable-pas-form';
```

Comment:
These validations also need to include the extra questions for the land use actions. E.g. if `dcpPfzoningauthorization > 0` then `dcpZoningauthorizationpursuantto` and `dcpZoningauthorizationtomodify` are required. That goes for all land use actions that have extra questions (listed out in `land-use-actions.js`

Reply:
Gotcha! Will need help on that...